Python/modsupport.c

```diff
@@ -710,5 +705,6 @@ PyModule_AddType(PyObject *module, PyTypeObject *type)
     const char *name = _PyType_Name(type);
     assert(name != NULL);
 
-    return PyModule_AddObjectRef(module, name, (PyObject *)type);
+    Py_INCREF(type);
+    return PyModule_Add(module, name, (PyObject *)type);
```

Comment:
I don't see the purpose of this change. IMO PyModule_AddObjectRef() is a saner API and should be preferred, when possible.

Reply:
Changes in this file were written several months ago, before adding PyModule_AddObjectRef.